src/app/core/config/config.service.spec.ts

```diff
@@ -550,4 +551,89 @@ describe("ConfigService", () => {
     const actualFromNew = service.getConfig("view:X");
     expect(actualFromNew).toEqual(newFormat);
   }));
+
+  it("should migrate to new photo dataType", fakeAsync(() => {
+    const config = new Config();
+    const oldFormat = {
+      attributes: {
+        myPhoto: {
+          dataType: "file",
+          editComponent: "EditPhoto",
+          label: "My Photo",
+        },
+        simpleFile: {
+          dataType: "file",
+          label: "Simple File attachment",
+        },
+      },
+    };
+
+    const newFormat: EntityConfig = {
+      attributes: {
+        myPhoto: {
+          dataType: "photo",
+          label: "My Photo",
+        },
+        simpleFile: {
+          dataType: "file",
+          label: "Simple File attachment",
+        },
+      },
+    };
+
+    config.data = { "entity:X": oldFormat };
+    updateSubject.next({ entity: config, type: "update" });
+    tick();
+    const actualFromOld = service.getConfig<EntityConfig>("entity:X");
+    expect(actualFromOld).toEqual(newFormat);
+
+    config.data = { "entity:X": newFormat };
+    updateSubject.next({ entity: config, type: "update" });
+    tick();
+    const actualFromNew = service.getConfig<EntityConfig>("entity:X");
+    expect(actualFromNew).toEqual(newFormat);
+  }));
+
+  fit("should migrate to Percentage dataType", fakeAsync(() => {
```

Comment:
```suggestion
  it("should migrate to Percentage dataType", fakeAsync(() => {
```

reactivate all tests before merge